Makefile

```diff
@@ -148,14 +159,26 @@ kustomize: ## Download kustomize locally if necessary.
 ############################################################
 # unit test
 ############################################################
+GOSEC = $(shell pwd)/bin/gosec
+GOSEC_VERSION = 2.9.6
 
 test:
-	go test -v ./...
+	go test $(TESTARGS) ./...
+
+test-coverage: TESTARGS = -json -cover -covermode=atomic -coverprofile=coverage_unit.out
+test-coverage: test
 
 test-dependencies:
 	curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v$(KBVERSION)/kubebuilder_$(KBVERSION)_$(GOOS)_$(GOARCH).tar.gz | tar -xz -C /tmp/
 	sudo mv /tmp/kubebuilder_$(KBVERSION)_$(GOOS)_$(GOARCH) /usr/local/kubebuilder
 
+gosec-install:
+	curl -L https://github.com/securego/gosec/releases/download/v$(GOSEC_VERSION)/gosec_$(GOSEC_VERSION)_$(GOOS)_$(GOARCH).tar.gz | tar -xz -C /tmp/
+	sudo mv /tmp/gosec $(GOSEC)
```

Comment:
Not needed, but FYI:

If it will be installed to a consistent location under the parent directory, you can use that as the target name. Then you can use that as a pre-req for other targets (like `gosec-scan` below), and it will only install it if it's not already present.

For example: https://github.com/JustinKuli/governance-policy-addon-controller/blob/main/Makefile#L151

Reply:
Ah, interesting. I think I had forgotten that about Makefiles that it could build dependencies only if they didn't exist. Makes me realize that what I know about Makefiles is probably a drop in an ocean!

Reply:
It's kind of funny, because the `go-get-tool` that kubebuilder (and others) use *also* only gets things if they don't already exist... it's like they re-built existing make functionality